### PR TITLE
#167974118 a user can signin

### PR DESCRIPTION
**Pull request by Bahatiroben**
#### What does this PR do?
it implements the sign in feature
#### Description of Task to be completed?
```
- the user can sign up, provided he has provided email and password that are already signed up
- a user gets an error when he tries to sign in an non existing account
- user gets error when they try to signin without or with incorrect email or password inclusively
```
#### How should this be manually tested?
```
- clone the repo
- run npm dev to start the server
- signup and then query the sign in endpoint as specified in the documentstion
```
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167974118
#### Screenshots (if appropriate)
Screen Shot 2019-08-22 at 14 44 12
Screen Shot 2019-08-22 at 14 46 53

#### Questions:
N/A

### CODE REVIEW

File: backend/services/userService.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ class UserService {
       throw error;
     }
   }
+
+  static async getUser(email) {
+    try {
+      const user = await database.Users.findOne({ where: { email } });
```
Comment (willywax) [inline backend/services/userService.js]: Let's do as we did in Products Service.js File.

import database from '../database/models';

const { Users } = database;

then we can use just Users.findOne(...

**Review by willywax — COMMENTED** [review]
Let's fix the UserService. Everything else Looks great

File: backend/services/userService.js (hunk 1 above)
Comment (Bahatiroben) [reply backend/services/userService.js]: thanks for suggesting the changes. You can check their implementation in an updated file.

**Review by drayzii — APPROVED** [review]
Looks good to me!

**Review by Dkabiswa — APPROVED** [review]
this actually works

**Pull request merged**